fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -28,4 +28,5 @@
       # queue limit is hit - 'block' will halt further reads and keep retrying to flush the
       # buffer to the remote - default is 'exception' because in_tail handles that case
       buffer_queue_full_action "#{ENV['BUFFER_QUEUE_FULL_ACTION'] || 'exception'}"
+      ssl_verify "#{ENV['ES_COPY_SSL_VERIFY'] || 'true'}"
```

Comment:
@richm, that's my understanding, too.
I'm working on the last item, now...
- we'll need to figure out another way to test without exposing ssl_verify